src/components/DocsNavigation/DocsNavigationMobile/DocsNavigationMobile.tsx

```diff
@@ -14,6 +14,7 @@ function DocsNavigationDesktop({ pathname }: { pathname: string; children?: Reac
       >
         <div className={styles.DocsPickerContainer}>
           <DocsPickerMobile path={pathname} />
+          {children}
```

Comment:
I cannot seem to find any usage of this in the PR. Will look into the code now, but if you know share the reason for this addition

Reply:
To make the filter tabs available on Mobile navbar, previously they were on only Desktop
<img width="381" alt="Screenshot 2024-10-23 at 4 14 02 PM" src="https://github.com/user-attachments/assets/626d55c7-10d0-49f8-80aa-6232d87c70c2">